src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -89,10 +89,19 @@ class Request
     /**
      * Request body parameters ($_POST).
      *
+     * @deprecated since Symfony 6.2
+     *
      * @var InputBag
      */
     public $request;
```

Comment:
this should probably be implemented through `__get` and `__set`, to avoid issue where someone re-assign the property, where we would have different instances in the new property and the old one (breaking the BC layer)